docs/spark/tutorials/get-started-windows-instructions.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Get Started with .NET for Apache Spark on Windows
+description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows.
+ms.date: 12/30/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on Windows.
+---
+
+# Tutorial: Get Started with .NET for Apache Spark on Windows
+
+This tutorial will show you how to run a .NET for Apache Spark app using .NET Core on Windows.
+
+> [!div class="checklist"]
+>
+> * Pre-requisites to run your .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run your .NET for Apache Spark application
+
+## Pre-requisites
+
+- Download and install the following: **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** | **[Visual Studio 2019](https://www.visualstudio.com/downloads/)** | **[Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)** | **[Apache Spark 2.4.1](https://archive.apache.org/dist/spark/spark-2.4.1/spark-2.4.1-bin-hadoop2.7.tgz)**
```

Comment:
either say following products or just list one by one

also links to VS downloads from docs should use campaign links
https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019

Reply:
Thanks for your comments! I have changed to `following products`. 
Can you please specify what you mean by `campaign links`? Thanks.